repo/packages/K/kibana/1200/marathon.json.mustache

```diff
@@ -0,0 +1,120 @@
+{
+  "id": "{{service.name}}",
+  "cpus": {{kibana.cpus}},
+  "mem": {{kibana.mem}},
+  "instances": 1,
+  "user": "{{service.user}}",
+  "container": {
+    "type": "MESOS",
+    {{#service.virtual_network_enabled}}
+    "portMappings": [
+      {
+        "name": "kibana",
+        "containerPort": 0,
+        "hostPort": 0,
+        "labels": {
+          "VIP_0": "/web.{{service.name}}:80"
+        }
+      },
+      {
+        "name": "proxy",
+        "hostPort": 0,
+        "containerPort": 0
+      }
+    ],
+    {{/service.virtual_network_enabled}}
+    "docker": {
+      "image": "{{resource.assets.container.docker.nginx}}"
+    }
+  },
+  "cmd": "./init.sh",
+  "constraints": {{kibana.placement}},
+  "labels": {
+    "DCOS_SERVICE_NAME": "{{service.name}}",
+    "DCOS_SERVICE_PORT_INDEX": "1",
+    "DCOS_SERVICE_SCHEME": "http",
+    "DCOS_SERVICE_REWRITE_REQUEST_URLS": "false",
+    "MARATHON_SINGLE_INSTANCE_APP": "true",
+    "DCOS_PACKAGE_FRAMEWORK_NAME": "{{service.name}}"
+  },
+  "env": {
+    "ELASTIC_VERSION": "6.8.1",
+    "FRAMEWORK_NAME": "{{service.name}}",
+    "FRAMEWORK_USER": "{{service.user}}",
+    "KIBANA_PLUGINS": "{{kibana.plugins}}",
+    "ELASTICSEARCH_URL": "{{kibana.elasticsearch_url}}",
+    {{#kibana.elasticsearch_tls}}
+    "KIBANA_ELASTICSEARCH_TLS": "{{kibana.elasticsearch_tls}}",
+    {{/kibana.elasticsearch_tls}}
+    "KIBANA_USER": "{{kibana.user}}",
+    "KIBANA_PASSWORD": "{{kibana.password}}",
+    {{#service.virtual_network_enabled}}
+    "VIRTUAL_NETWORK_ENABLED": "true",
+    {{^service.virtual_network_plugin_labels.isEmpty}}
+    "VIRTUAL_NETWORK_PLUGIN_LABELS": "{{#service.virtual_network_plugin_labels}}{{key}}:{{value}},{{/service.virtual_network_plugin_labels}}",
+    {{/service.virtual_network_plugin_labels.isEmpty}}
+    {{/service.virtual_network_enabled}}
+    "CUSTOM_YAML_BLOCK_BASE64": "{{kibana.custom_kibana_yml}}"
+  },
+  "fetch": [
+    {
+      "uri": "{{resource.assets.uris.kibana-tar-gz}}",
+      "cache": true
+    },
+    {
+      "uri": "https://infinity-artifacts-ci.s3.amazonaws.com/autodelete7d/kibana/20200601-114631-66kytXDpebf1yVYj/nginx.conf.tmpl",
```

Comment:
This URI has a `autodelete7d` filed and will expire, we cannot release with this, same for other one below.
Can you check if the stubs we used to create this also had these `autodelete7d` fields? We'll need to verify if our tooling didn't catch instances of this.

Reply:
Previous releases reference resource.json assets:
https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/K/kibana/700/marathon.json.mustache#L59-L73

https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/K/kibana/700/resource.json#L2-L7

Reply:
The stub used was permanent only but somehow some random artifacts got rendered in the marathon as the version was old and its corresponding artifacts were already uploaded.

Reply:
There is no need to worry as the resource.json has correct URLs. I have removed the rendered uris. Should be OK now.